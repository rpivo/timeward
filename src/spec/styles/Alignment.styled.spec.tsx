import React from 'react';
import renderer from 'react-test-renderer';
import StyledAlignment from '@styles/components/Alignment.styled';

describe('Button', () => {

  const styledAlignment = <StyledAlignment />;

  describe('render', () => {
    const testRenderer = renderer.create(styledAlignment);

    it('should render correctly', () => {
      testRenderer.toJSON();
      expect(testRenderer).toMatchSnapshot();
    });

  });
});
