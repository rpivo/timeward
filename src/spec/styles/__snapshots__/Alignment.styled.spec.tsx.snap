// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button render should render correctly 1`] = `
<div
  className="sc-AxjAm hMrxYV"
/>
`;
